packages/ava/src/advisor/advise-pipeline/advicesForChart.ts

```diff
@@ -37,15 +37,23 @@ export function advicesForChart(
     return null;
   }
 
+  const defaultDataProps = dataFrame.info();
   // get dataProps from dataframe
   let dataPropsForAdvice: BasicDataPropertyForAdvice[];
   if (dataProps) {
     // filter out fields that are not included for advising
     dataPropsForAdvice = fields
       ? dataProps.filter((dataProp: BasicDataPropertyForAdvice) => fields.includes(dataProp.name))
       : dataProps;
+    dataPropsForAdvice.map((item) => {
```

Comment:
what is this for?

and Array.map would not change itself at all

Reply:
removed